docs/cmd/kumactl/kumactl_get_zone-egress.md

```diff
@@ -0,0 +1,33 @@
+## kumactl get zone-egress
+
+Show a single ZoneEgress resource
+
+### Synopsis
+
+Show a single ZoneEgress resource.
+
+```
+kumactl get zone-egress NAME [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for zone-egress
+```
+
+### Options inherited from parent commands
+
+```
+      --api-timeout duration   the timeout for api calls. It includes connection time, any redirects, and reading the response body. A timeout of zero means no timeout (default 1m0s)
```

Comment:
`1m0s` ? Is that 1 minute, 0 seconds? or should that read `1ms` ?

Reply:
Actually, what is the unit of the timeout duration?

Reply:
it's `1m`, it's the default way of how go is rendering `time.Minute`